web/src/main/java/com/epam/esm/contollers/ControllerAdvice.java

```diff
@@ -0,0 +1,48 @@
+package com.epam.esm.contollers;
+
+import com.epam.esm.exception.EntityNotFoundException;
+import com.epam.esm.exception.IncorrectEntityException;
+import com.epam.esm.impl.ExceptionResponse;
+import com.epam.esm.validator.ValidationError;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@RestControllerAdvice
+public class ControllerAdvice {
+    @ExceptionHandler(IncorrectEntityException.class)
+    public ResponseEntity<ExceptionResponse> handlerException(IncorrectEntityException exception) {
+        return new ResponseEntity<ExceptionResponse>(
+                new ExceptionResponse(
+                        exception.getMessage(),
+                        enumListToStringList(exception.getValidationErrors()),
+                        LocalDateTime.now().toString(),
+                        "Error: 0001")
```

Comment:
error codes can be moved to constants as well

Reply:
Fixed